assets/prefabs/Equipment/Iron/IronEspeebastarde.prefab

```diff
@@ -0,0 +1,23 @@
+{
+    "parent" : "engine:iconItem",
+    "Item" : {
+        "icon" : "Equipment:IronEspeebastarde",
```

Comment:
This and the icon name do not match (an extra e here). Make the 'b' in bastard capital at both places too while you're at it.